src/MoBi.Presentation/Presenter/ExtendablePathAndValueBuildingBlockPresenter.cs

```diff
@@ -33,9 +33,9 @@ public abstract class ExtendablePathAndValueBuildingBlockPresenter<TView, TPrese
    {
       protected readonly IPathAndValueEntityToPathAndValueEntityDTOMapper<TPathAndValueEntity, TStartValueDTO> _valueMapper;
 
-      private readonly IInteractionTasksForExtendablePathAndValueEntity<TBuildingBlock, TPathAndValueEntity> _interactionTasksForExtendablePathAndValueEntity;
+      protected readonly IInteractionTasksForExtendablePathAndValueEntity<TBuildingBlock, TPathAndValueEntity> _interactionTasksForExtendablePathAndValueEntity;
       protected BindingList<TStartValueDTO> _startValueDTOs;
-      private readonly IEmptyStartValueCreator<TPathAndValueEntity> _emptyStartValueCreator;
+      protected readonly IEmptyStartValueCreator<TPathAndValueEntity> _emptyStartValueCreator;
```

Comment:
Can these two go back to private since you pulled up the code for `AddNewPathAndValueEntity`?